cocos/network/WebSocket.h

```diff
@@ -40,6 +40,10 @@
 #include "platform/CCPlatformMacros.h"
 #include "platform/CCStdC.h"
 
+#if CC_TARGET_PLATFORM == CC_PLATFORM_ANDROID
+#include "platform/android/jni/JniHelper.h"
+#endif
+
```

Comment:
Can remove platform specific codes here?

Reply:
JNI types `jstring` `jlong` are referenced in friend function, these lines should be kept.